### ISSUE TITLE
Has this project been abandoned?

### ISSUE THREAD

**Issue by maettu-this** [dejavu-fonts/dejavu-fonts#379]
I have requested #339, initially 9 years ago at https://bugs.freedesktop.org/show_bug.cgi?id=58444, never got opened nor anything else, neither did the mentioned GitHub equivalent #339. Seeing no commit for 4 years here, seeing several pull requests pending, seeing issues not being managed... @moyogo or somebody else, please give an update, or state that this project is no longer active.

**Comment by kimmus** [comment]
Crickets.
;)
I think the silence answers your question, unfortunately.

**Comment by rknop** [comment]
That's so sad.  I really like the DejaVu fonts.  There's a sans, a serif, and a mono font that all scale well together, and I like the way they look.  Plus, at least as of five years ago, it had pretty good Unicode coverage.

I'll keep using them for now, but eventually I'll probably have to move on as the standard Unicode list of defined symbols grows and DejaVu doesn't keep up.  Has anybody made a fork that is maintained?  What are you all moving to?  Ideally, I'd like to find a continuation of DejaVu that keeps the same fonts and just maintains them; I like the look of the fonts and don't want something completely new.

**Comment by maettu-this** [comment]
Just recently did an evaluation of potential alternatives:

- Source Code Pro
- Cascadia (new with Visual Studio 2022)
- Consolas
- Lucida Console
- JetBrains Code/Mono
- ...

https://fonts.google.com/?category=Monospace gives another nice overview of potential alternatives. Source Code Pro and JetBrains Code/Mono can be found there as well. I personlly favor **Source Code Pro**.

**Comment by kimmus** [comment]
I used Dejavu Mono until I needed glyphs that it doesn’t have and which now seem unlikely to be added. I now use Liberation Mono at https://github.com/liberationfonts/liberation-fonts

**Comment by lilydjwg** [comment]
> the standard Unicode list of defined symbols grows and DejaVu doesn't keep up.

This doesn't bug me. fontconfig will just fallback to other fonts that contains the missing characters so I'm still preferring DejaVu.

**Comment by GlitterHorn** [comment]
> > the standard Unicode list of defined symbols grows and DejaVu doesn't keep up.
> 
> This doesn't bug me. fontconfig will just fallback to other fonts that contains the missing characters so I'm still preferring DejaVu.

I can't find any free Unicode fonts at the moment with the complete set of runic characters, 16A0-16FF. I was apparently falling back to Arial Unicode MS, which was removed from my machine in an office update at some point, and now I have no font with a complete character set. Are there any current fonts with this set?

**Comment by kimmus** [comment]
These fonts on my system have the runic characters: Cardo, Catrinity, Noto Sans Runic, Quivira, Segoe UI Historic. Of those, at least the first four are free. The Segoe fonts may have come bundled with the operating system and/or some application—I haven’t paid for them, at least not separately.

Edit: Cardo actually has very few runic characters. Segoe UI Historic has most but not all. The other three have the complete block.

**Comment by iandoug** [comment]
See (selected) fonts with wide coverage:

https://www.keyboard-design.com/academic-font-coverage-comparison.html

**Comment by pglpm** [comment]
I suspected as much... So sad to see this great project has been abandoned.

**Comment by ticoeteco23gb** [comment]
So sad to see another great project discontinued... we could at least have some kind of reason for the lack of updates.

**Comment by TimurGerasimov250** [comment]
iosevka is a great choice

**Comment by pglpm** [comment]
> iosevka is a great choice

It's very interesting, but it's still a long shot from being what DejaVu is. It's not quite a proportional font, it doesn't have a serif/sans distinction, and I think DejaVu still covers many more glyphs.

**Comment by TimurGerasimov250** [comment]
No. Iosevka 33.2.8 (latest version) has over 8700 gltphs and 7500 characters (in fixed)
DejaVu fonts have much less, dejavu sans has 6,253 glyphs in regular and over 5,900 characters.
Interesting!

**Comment by Logan-Quest** [comment]
I'm devastated. This was always one of the first things I installed on a new system. Such a great set of fonts.
